### PR TITLE
Adding LSS to registry.json

### PR DESCRIPTION
**Pull request by traddacharlie**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1128,5 +1128,20 @@
...
+    "poolId": "4d470d31f80c01d797996b43abdfe7339ae5b408fb31cc655a80ccd7",  
+    "social": {                                                           
+      "facebook": "", 
+      "twitter": "https://twitter.com/lss_cardanopool",
+      "website": "https://longshortsignal.com/cardanopool",  
+      "reddit": "",  
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "lss_cardanopool",
```
Comment (leo42) [inline registry.json:1143]: Please use a personal telegram handle.

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (traddacharlie) [reply registry.json]: Hello I have just made the change accordingly, please let me know if any issues.

**Review by leo42 — APPROVED** [review]

**Comment by leo42** [comment]
Looks good. Welcome to CSPA :)

**Comment by traddacharlie** [comment]
Thank you so much, it's an honor and we are fully committed to only running one pool and therein SPA :-)

**Pull request merged**